packages/validator/src/services/attestationDuties.ts

```diff
@@ -371,6 +371,28 @@ export class AttestationDutiesService {
       .catch((e: Error) => {
         this.logger.error("Failed to redownload attester duties when reorg happens", logContext, e);
       });
+
+    const beaconCommitteeSubscriptions: routes.validator.BeaconCommitteeSubscription[] = [];
+    const epochDuties = this.dutiesByIndexByEpoch.get(dutyEpoch)?.dutiesByIndex;
+
+    if (epochDuties) {
+      for (const {duty, selectionProof} of epochDuties.values()) {
+        beaconCommitteeSubscriptions.push({
+          validatorIndex: duty.validatorIndex,
+          committeesAtSlot: duty.committeesAtSlot,
+          committeeIndex: duty.committeeIndex,
+          slot: duty.slot,
+          isAggregator: selectionProof !== null,
+        });
+      }
+    }
+
+    if (beaconCommitteeSubscriptions.length > 0) {
+      const subscriptionsBatches = batchItems(beaconCommitteeSubscriptions, {batchSize: SUBSCRIPTIONS_PER_REQUEST});
+      await Promise.all(
+        subscriptionsBatches.map((subscriptions) => this.api.validator.prepareBeaconCommitteeSubnet({subscriptions}))
+      );
+    }
```

Comment:
this is missing the asserts to make sure all requests succeeded, otherwise this could hide errors

```ts
      for (const res of responses) {
        res.assertOk();
      }
```

this is the same logic as we already have here

https://github.com/ChainSafe/lodestar/blob/5317389489e1553be2e98f79f0c0f8c74ff79f28/packages/validator/src/services/attestationDuties.ts#L222-L226

it makes sense to have a private method for this to make sure batching / error handling is the same